src/mongo/client/dbclient.cpp

```diff
@@ -1416,6 +1424,10 @@ bool DBClientWithCommands::eval(const string& dbname,
 }
 
 bool DBClientWithCommands::eval(const string& dbname, const string& jscode) {
```

Comment:
i think you have a good point, I'm going to change it since this seems like actually a pretty reasonable place to inline